src/random.ts

```diff
@@ -98,15 +98,17 @@ export class Random {
    * Returns random element from the given array.
    *
    * @template T The type of the entries to pick from.
-   * @param array Array to pick the value from. Defaults to `['a', 'b', 'c']`.
+   * @param array Array to pick the value from.
    *
    * @example
-   * faker.random.arrayElement() // 'b'
    * faker.random.arrayElement(['cat', 'dog', 'mouse']) // 'dog'
+   * faker.random.arrayElement([0, 1, 2, 3]) // 3
    */
-  arrayElement<T = string>(
-    array: ReadonlyArray<T> = ['a', 'b', 'c'] as unknown as ReadonlyArray<T>
-  ): T {
+  arrayElement<T>(array: ReadonlyArray<T>): T {
+    if (typeof array === 'undefined' || array === null || array.length <= 0) {
```

Comment:
This can be shorten to
```suggestion
    if (array == null || array.length === 0) {
```

This does exactly the same but is much easier to read and shorter

Reply:
I would argue about being easier to read because you use a loosely type check `==` for bullish values and a strong type check for the array length. Furthermore, I'm a friend of abandoning the loose type check operator since it brings consistency into a project (and somewhat safety over only using `==` as a consistent check).

When on that point? Why do you use `null` as compare value instead of the more often used `undefined`? Would also do the same.

Reply:
That said, I'm not saying you are wrong. Just want to give you my opinion on that toppic.

Reply:
Using ` == null ` is a really common pattern in js.
I think there is even a eslint rule for that 👀 
Yeah https://eslint.org/docs/rules/eqeqeq
We can configure it like my suggested way

It's a special case when you want to compare something is `null` or `undefined`, then use `== null`

Reply:
The `eqeqeq` eslint rule is exaclty what im referring to as well. Since it disallows the loose type check operation `==` and requires you to use the strong one `===`.
But I guess you referring to the `'smart'` configuration option instead of `'always'`.

Reply:
Having an eslint rule for that is probably good for consistency.

Reply:
I might be biased but I rarely see `== null`. `== undefined` is what I come across all the time.